### DISCUSSION TITLE
Linux 平台安装 QQ NT 通过 LiteLoaderQQNT 的方式使用机器人，不定期退出登录

### DISCUSSION THREAD

**Discussion by fred913** [NapNeko/NapCatQQ#382] [Q&A] [answered]
as title

退出登录并无征兆，直接显示 Kicked，GUI 界面提示 登录已失效。

**Comment by MliKiowa** [comment, marked as answer]
尝试切换QQ账号

**Reply by fred913** [reply]
会和手机端登录（手机端是否登录、手机端环境）有关系吗？现在是扫码登录，据您了解切换到帐密登录是否会有帮助？

**Reply by MliKiowa** [reply]
> 会和手机端登录（手机端是否登录、手机端环境）有关系吗？现在是扫码登录，据您了解切换到帐密登录是否会有帮助？

跟账号和设备有关系

**Reply by fred913** [reply]
感谢回复，我测试2-3天再反馈
